jerry-core/ecma/base/ecma-globals.h

```diff
@@ -1092,6 +1092,11 @@ typedef double ecma_number_t;
 #define ECMA_NUMBER_FRACTION_WIDTH   (52)
 #endif /* !ENABLED (JERRY_NUMBER_TYPE_FLOAT64) */
 
+/**
+ * Value 'NaN' of ecma_number_t
+ */
+#define ECMA_NUMBER_NAN (ecma_number_t) (0.0 / 0.0)
```

Comment:
It seems MSVC doesn't like 0.0/0.0 to generate NaN:
C:/projects/jerryscript/jerry-core/ecma/operations/ecma-conversion.c(935): error C2124: divide or mod by zero [C:\projects\jerryscript\build\jerry-core\jerry-core.vcxproj]

But you can use the ecma_number_make_nan() function which is used everywhere to generate NaN.

Reply:
Was trying to avoid using that function. Seems like I need to use that for now. Thank you. :)